duckduckgo/macos-browser
# Use latest privacy dashboard

Task/Issue URL:
Tech Design URL:
CC: @tomasstrba @mallexxx @samsymons

**Description**:
Updates Privacy Dashboard with some small fixes from product review.
 - https://github.com/more-duckduckgo-org/duckduckgo-privacy-dashboard/pull/17
 - https://github.com/more-duckduckgo-org/duckduckgo-privacy-dashboard/pull/18
 - https://github.com/more-duckduckgo-org/duckduckgo-privacy-dashboard/pull/19

**Steps to test this PR**:
General copy changes (see PRs linked above for details)

Bugfix on blocked company count
1. Go to https://uk.yahoo.com/
2. The number of companies blocked in the tracker summary should be correct (ignoring first party company in the count)

Company icons and trackers sorted by prevalence
1. Go to https://www.theguardian.com
2. The company icons should be sorted by prevalence (Google, then Fastly)
3. Click into the tracker details -- the track groups should be sorted by company prevalence (Google, then Fastly even though Fastly has more trackers on the page)

View updates on protection toggles
1. Go to https://cheese.com
2. Toggle off protections -- previously there was a flash as the popover webview refreshed, this is now gone with popover state update correctly and views re-rendering

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
